CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Icinga DB Changelog
 
+## 1.1.2 (2024-04-11)
+
+This release addresses multiple issues related to fault recoveries,
+with a particular focus on retryable database errors that may occur when using Icinga DB with database clusters.
+
+* Retry broken pipe errors (`EPIPE`). #693
+* Retry every database error. #698
+* MySQL/MariaDB: Use strict SQL mode. #699
+* Try setting `wsrep_sync_wait` for MySQL/MariaDB connections. #711
+* history.Sync: Don't operate on closed channel. #725
+* Flatten(): render even large numbers as-is, not using scientific notation. #657
+* Enhance HA "Taking over", "Handing over" logging. #692
+* Update dependencies. #649 #650 #658 #661 #662 #663 #664 #670 #673 #675 #678 #680 #682 #687 #696 #701 #702 #703 #704 #705 #706 #712 #715 #720 #721 #727 #728 #733
```

Comment:
It's done like last time, but that PR list is very long and doesn't look helpful at all. Maybe just link to https://github.com/Icinga/icingadb/pulls?q=is%3Apr+milestone%3A1.1.2+label%3Adependencies instead?